src/components/Close/CloseSUTable.jsx

```diff
@@ -244,14 +283,16 @@ class CloseSUTable extends React.Component {
                                 (pagination.page - 1) * pagination.size,
                                 Math.min(pagination.page * pagination.size, displayedLines.length),
                               )
-                              .map((line) => (
-                                <CloseSurveyUnitLine
-                                  key={line.id}
-                                  lineData={line}
-                                  isChecked={checkboxArray[line.id]}
-                                  updateFunc={() => toggleCheckBox(line.id)}
-                                />
-                              ))}
+                              .map((line) => {
+                                const element = checkboxArray.filter((element) => element.id === line.id)[0]
```

Comment:
```suggestion
                                const element = checkboxArray.find((checkbox) => checkbox.id === line.id);

```
Array.find( criteria ) return first matching element